Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/Dictionary.java

```diff
@@ -0,0 +1,7 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+public interface Dictionary {
+	
+	public String getRandomWord();
```

Comment:
The `public` modifier here is redundant because methods declared in an interface have to be public